docs/src/pages/guides/api/api.md

```diff
@@ -43,7 +43,7 @@ We avoid documenting native properties supported by the DOM like [`className`](/
 
 ### CSS Classes
 
-All components accept a [`classes`](/customization/how-to-customize/#overriding-styles-with-classes) prop to customize the styles.
+All components accept a `classes` prop to customize the styles.
```

Comment:
I think we should rather evaluate what this section is for:
1. do we still need it?
1. where did the `overriding-styles-with-classes` section go
1. why was the codeblock below removed

Again, please use accurate PR titles and description so that people can follow changes at a glance and not have to check each line changed to catch unrelated changes.

Reply:
1. I don't think that it's relevant to the problem I'm going after, the 404, these changes are cleanup for outdated content.
2. We have removed it, we don't want developers to use this approach.
3. It's about makeStyles, which is legacy.

Reply:
I have reverted